### PR TITLE
Finally fixes secure storage bullshit

### PR DESCRIPTION
**Pull request by Spacedong**
red alert safes on escape pods are no longer openable without it being red alert, fixes issue #2597

### CODE REVIEW

File: html/changelogs/Spacedong - securefix.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
-################################
```
Comment (Gandalf2k15) [inline html/changelogs/Spacedong - securefix.yml:1]: what?

Comment (Spacedong) [inline html/changelogs/Spacedong - securefix.yml]: /shrug

Comment (Gandalf2k15) [inline html/changelogs/Spacedong - securefix.yml:1]: fix it!

Comment (Spacedong) [inline html/changelogs/Spacedong - securefix.yml]: done!!

**Comment by Carbonhell** [comment]
c o n f l i c t s

**Comment by ArcLumin** [comment]
ｃｏｎｆｌｉｃｔｓ，　ｌａｄｓ

**Comment by Spacedong** [comment]
i cant fix them fuck

**Pull request closed without merging**